core/wiki/macros/list.tid

```diff
@@ -22,7 +22,7 @@ tags: $:/tags/Macro
 <$action-listops $tiddler=<<targetTiddler>> $field=<<targetField>> $subfilter="+[insertbefore<actionTiddler>,<currentTiddler>]"/>
 \end
 
-\define list-links-draggable(tiddler,field:"list",emptyMessage,type:"ul",subtype:"li",class:"",itemTemplate)
+\define list-links-draggable(tiddler,field:"list",emptyMessage,type:"ul",subtype:"li",class:"",itemTemplate, field:"")
```

Comment:
here the default needs to be `caption` for backwards compatibility reasons

Reply:
There already is a `field:"list"` parameter at the second position.

Reply:
@kookma could you please resolve this issue with the duplicate field parameter? Thank you.

Reply:
It looks like we've still got two parameters called "field"